### ISSUE TITLE
Update to the design principle language?

### ISSUE THREAD

**Issue by koalie** [w3c/idcg#26]
cf. #24.

I'd like us to consider Tobie's feedback (which is in accordance to why in drafting the BLM statement @wareid @LJWatson and @koalie chose not to reuse it as is) and update the "Web for All" [design principle](https://www.w3.org/Consortium/mission#principles), and any other part of this page that needs a revision.

Help wanted.

I propose to then submit our revision to the W3C team for implementation.

**Comment by wareid** [comment]
I can help! Agreed that it needs updating. The spirit is there, the language needs updating :).

**Comment by LJWatson** [comment]
+1 to submitting our revised edition with the edit proposed by Tobie.

**Comment by koalie** [comment]
[Tobie's proposal](https://github.com/w3c/idcg/issues/24#issuecomment-661801816)

**Comment by koalie** [comment]
> +1 to submitting our revised edition with the edit proposed by Tobie.

Status update:
I ran the proposed revision by the Team on August 28, 2020. All 5 team members who weighed in between August 28 and September 4 did so to push back.

The reasons given were of the type:

- It's important to NOT remove "physical or mental ability": W3C's inclusiveness with regard to accessibility is well-known and well-regarded. Many people are not even cognizant of disability as an aspect of diversity and inclusion, so the prompt to think about it and address it has a significant impact.
- The rewrite to modernize the statement was viewed as missing important things and putting too much emphasis on others. 
- It was suggested that the combined use of the terms 'contribution and benefit' may infer that if you contribute, you therefore benefit, and could lead to some thorny questions. So "'making the benefit of the web available.. regardless of x,y,z'" was preferred.


I propose to close this issue for lack of consensus.

**Comment by torgo** [comment]
Discussed on call 12-01-2020 - and agreed to keep this open and continuing to work on a re-write of the principle.

**Comment by LJWatson** [comment]
I agree that the principle should be updated. The phrase "physical and 
mental ability" feels very out-dated in particular.
